Meta/gn/secondary/AK/BUILD.gn

```diff
@@ -287,3 +287,12 @@ write_cmake_config("ak_debug_gen") {
     "XML_PARSER_DEBUG=",
   ]
 }
+
+write_cmake_config("ak_backtrace_gen") {
+  input = "Backtrace.h.in"
+  output = "$root_gen_dir/AK/Backtrace.h"
+  values = [
+    "Backtrace_FOUND=",
+    "Backtrace_HEADER=",
```

Comment:
on GNU/Linux and macOS, we want this to be

```
  values = [
    "Backtrace_FOUND=1",
    "Backtrace_HEADER=execinfo.h",
  ]
```

Essentially the original CMake config was just a wrapper around `find_package(Backtrace)` for musl linux and other BSDs that need `libexecinfo` (<https://cmake.org/cmake/help/latest/module/FindBacktrace.html>)

Reply:
Making that change requires updating write_cmake_config.py line 87 to say `except ValueError:` or `except:` rather than `except _:` though, my system python was complaining.